pkg/transaction/logeventbuilder.go

```diff
@@ -84,7 +84,7 @@ type SummaryBuilder interface {
 	SetProxy(proxyID, proxyName string, proxyRevision int) SummaryBuilder
 	SetRunTime(runtimeID, runtimeName string) SummaryBuilder
 	SetEntryPoint(entryPointType, method, path, host string) SummaryBuilder
-
+	SetIsIncludedInUsageMetricEvent(isIncludedInUsageMetricEvent bool) SummaryBuilder
```

Comment:
Is this builder method still necessary if we are only concerned with the attribute being set when the new summary builder is instantiated?

Reply:
I thought its a standard to have the builder method for all the attributes, because I saw that there is a builder method for fields like APICDeployment, EnvironmentID etc as well that are only set when the new summary builder is instantiated

Reply:
That makes sense, then we can keep it and eventually clean up the extra setters.